### PR TITLE
Feature#15 code refactoring

### LINKED ISSUE #15: Code refactoring
# REQUIREMENTS
Find the bugs in CodeClimate and fix them.

### PR DESCRIPTION
**Pull request by daniel-yocupicio**
# Description
All code was refactored to eliminate codeclimate issues, some components were modified so that they do not exceed 25 lines and adapted so that the firebase clientid is not directly in the code.
   
  ## Screenshots 
test with jest, you already had a previous test but it was adapted to work with the refactored code. 
image

Some screenshots of the views that the app has:
Sign Up.
image

Log in
image

Flight list
image

Add Flight
image
image
image
image
image

A flight was added
image

### CODE REVIEW

**Review by ShamuXX — DISMISSED** [review]

**Review by LizyMiGo — DISMISSED** [review]

File: ReservacionDeVuelos/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+CLIENT_ID=959127818437-tk3msn40d4lslgkrm57a8dp4sti7fl8l.apps.googleusercontent.com
```
Comment (YilKanda) [inline ReservacionDeVuelos/.env:1]: This file should not appear on GitHUb, instead put an .env.example file.

File: ReservacionDeVuelos/src/components/molecules/CardFlight.js [hunk 2]
```diff
@@ -3,25 +3,23 @@ import {View} from 'react-native';
...
+export default CardFlight;
+
+/*
+        <Countries
         avDestino={avDestino}
         avOrigen={avOrigen}
         destino={destino}
         origen={origen}
       />
       <FlightDetails fecha={fecha} passengers={passengers} />
```
Comment (YilKanda) [inline ReservacionDeVuelos/src/components/molecules/CardFlight.js:24]: Do not leave on GitHub in comments. If you don't use it, remove it.

File: ReservacionDeVuelos/src/consts/globalstyles.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+/*
+  marginTop:
+    Dimensions.get('screen').height <= 640
+      ? {marginTop: 7}
+      : Dimensions.get('screen').height >= 641 &&
+        Dimensions.get('screen').height < 730
+      ? {marginTop: 15}
+      : {marginTop: 18},
+*/
```
Comment (YilKanda) [inline ReservacionDeVuelos/src/consts/globalstyles.js:25]: Do not leave on GitHub in comments. If you don't use it, remove it.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: ReservacionDeVuelos/.gitignore [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # OSX
 #
 .DS_Store
+.env
```
Comment (YilKanda) [inline ReservacionDeVuelos/.gitignore:4]: Only you need to do is place the .env.example file

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by ShamuXX — DISMISSED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by LizyMiGo — APPROVED** [review]

**Review by ShamuXX — APPROVED** [review]

**Review by LizyMiGo — APPROVED** [review]

**Pull request merged**